client/state/billing-transactions/actions.js

```diff
@@ -33,3 +36,26 @@ export const requestBillingTransactions = () => {
 			} );
 	};
 };
+
+export const sendBillingReceiptEmail = ( receiptId ) => {
```

Comment:
If you want to experiment early though, we can probably avoid having to store any isRequesting info in state.